scripts/filter.js

```diff
@@ -1,10 +1,13 @@
-const filters = { text: "", desc: "" };
+const filters = { text: "", desc: "", planet: 0};
```

Comment:
wait, what's the type of planet? idk like serpulo is 1 and erekir is 0? or like is this just to config if you want planet filtering or not.

Reply:
0 is no filter, 1 is serpulo, 2 is erekir, 3 is mixtech